docs/topics/null-safety.md

```diff
@@ -143,6 +143,17 @@ is `null`, the assignment is skipped and the expression on the right is not eval
 person?.department?.head = managersPool.getManager()
 ```
 
+## Nullable receiver
+
+Functions could be extended to use a [nullable receiver](extensions.md#nullable-receiver).
+This way you can check for the null condition and have a default defined in one place. 
+
+For example, the [`toString()` function](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to-string.html) can be called with a nullable receiver and it returns the "null" string value (not `null`):
+
+```kotlin
+null.toString() // returns "null"
```

Comment:
Let’s maybe add an example that introduces an explicit nullable type like “String?” here. Free-standing null-literals are kind of icky (and we don’t want to invite people to break their brain by doing things like `null + null`, which is senseless code anyway 😉 (https://youtu.be/wwplVknTza4)

Reply:
I tried to simplify the bug I created by not understanding this and boiled it down to an example, let me know what you think: [dee3b18](https://github.com/JetBrains/kotlin-web-site/pull/3133/commits/dee3b18d426b6f05ae9851974e57c3c4b955e3e6) and [8f1a862](https://github.com/JetBrains/kotlin-web-site/pull/3133/commits/8f1a8629e2dda5c52c48dccd489ca972ce1f2113)